contrib/redwood/plugins.yml

```diff
@@ -24,3 +24,11 @@
   description: |
     Tutor plugin to install Meilisearch and enable course content search in Studio.
     See the README on GitHub for further setup instructions.
+
+- name: k8s_harmony
+  src: -e 'git+https://github.com/openedx/openedx-k8s-harmony.git#egg=tutor-contrib-harmony-plugin&subdirectory=tutor-contrib-harmony-plugin'
+  url: https://github.com/openedx/openedx-k8s-harmony
+  author: 
+  maintainer: 
```

Comment:
We do need author and maintainer fields, such that people know who to contact to when they face an issue.

Reply:
@bradenmacdonald can you answer this one?

Reply:
For Author can you put "eduNEXT, OpenCraft, and contributors"
For maintainer, can we just link to https://github.com/orgs/openedx/teams/openedx-k8s-harmony-maintainers ? If not, list the three of us.